Resources/Prototypes/Entities/Objects/Devices/cartridges.yml

```diff
@@ -17,14 +17,11 @@
   - type: NotekeeperCartridge
```

Comment:
this should not be present in every other cartridge, make an abstract base cartridge and use that for them all